### PR TITLE
Tested and made revisions to Grafana create dashboard/panel tutorial.

### PR DESCRIPTION
**Pull request by jayercule**
# Description

* Test tutorials/OLD-grafana/create-dashboard-and-panel.md and rewrote where appropriate
* Updated instructions for new Grafana Cloud UI and took new screenshots
* Made small grammatical/flow edits to text

# Links

Updated images uploaded to shared Ercule/Timescale folder: https://drive.google.com/drive/folders/1UUY8SadUPqg6tP2oyWhMo4HqjX_kxW0S

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/about/latest/contribute-to-docs/)

# Review checklists

Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

*   [ ] Is the content technically accurate?
*   [ ] Is the content complete?
*   [ ] Is the content presented in a logical order?
*   [ ] Does the content use appropriate names for features and products?
*   [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

*   [ ] Is the content free from typos?
*   [ ] Does the content use plain English?
*   [ ] Does the content contain clear sections for concepts, tasks, and references?
*   [ ] Have any images been uploaded to the correct location, and are resolvable?
*   [ ] If the page index was updated, are redirects required
      and have they been implemented?
*   [ ] Have you checked the built version of this content?

### CODE REVIEW

File: tutorials/OLD_grafana/create-dashboard-and-panel.md [hunk 1]
```diff
@@ -20,53 +20,34 @@ and visualize data.
 Before you begin, make sure you have:
 
 *   [Installed TimescaleDB][install-timescale].
```
Comment (contentana) [inline tutorials/OLD_grafana/create-dashboard-and-panel.md:22]: Add instructions to cloud. "installed TimescaleDB or created a Timescale account (with link to sign up)."

**Review by contentana — CHANGES_REQUESTED** [review]
Hi, Jay! Please add the "create a Timescale account" option at the start.

@jonatas or @jamessewell , can one of you do a technical check on this tutorial? This is one of the bunch we're restoring for the blog! Thanks!

**Review by jonatas — APPROVED** [review]
From the technical perspective nothing changed, and this elements uses very basics of grafana.  Looks good to me but I haven't tested locally.

**Comment by contentana** [comment]
Thanks, @jonatas ! I really appreciate it.

File: tutorials/OLD_grafana/create-dashboard-and-panel.md (hunk 1 above)
Comment (contentana) [reply tutorials/OLD_grafana/create-dashboard-and-panel.md:22]: @jayercule, besides adding the cloud info, this is okay. Can you please make those changes, or can I go ahead and make them? I'll then just copy the md file to our blog, as agreed with Justin.

**Pull request closed without merging**